Kernel/Devices/DiskPartition.h

```diff
@@ -14,6 +14,11 @@ class DiskPartition final : public DiskDevice {
     virtual bool read_blocks(unsigned index, u16 count, u8*) override;
     virtual bool write_blocks(unsigned index, u16 count, const u8*) override;
 
+    // ^BlockDevice
+    virtual ssize_t read(FileDescription&, u8*, ssize_t) override{ return 0; };
```

Comment:
Missing a space character after override in these.
Also there's no need for `;` at the end of the line.